python/ray/_private/runtime_env/working_dir.py

```diff
@@ -147,6 +153,41 @@ def get_uris(self, runtime_env: "RuntimeEnv") -> List[str]:  # noqa: F821
             return [working_dir_uri]
         return []
 
+    async def _unpack_wheel(
+        self,
+        wheel_uri: str,
+        target_uri: str,
+        logger: Optional[logging.Logger] = default_logger,
+    ):
+        """Download and install a wheel URI, and then delete the local wheel file."""
+        module_dir = get_local_dir_from_uri(target_uri, self._resources_dir)
+
+        pip_install_cmd = [
+            "pip",
+            "install",
+            wheel_uri,
+            f"--target={module_dir}",
+        ]
+
+        logger.info(
+            "Running py_modules wheel install command: %s", str(pip_install_cmd)
+        )
+        try:
+            # TODO(architkulkarni): Use `await check_output_cmd` or similar.
+            exit_code, output = exec_cmd_stream_to_logger(pip_install_cmd, logger)
+        finally:
+            if Path(wheel_uri).exists():
+                Path(wheel_uri).unlink()
+
+            if exit_code != 0:
+                if Path(module_dir).exists():
+                    Path(module_dir).unlink()
+                raise RuntimeError(
+                    f"Failed to install py_modules wheel {wheel_uri}"
+                    f"to {module_dir}:\n{output}"
+                )
+        return module_dir
+
```

Comment:
Is this the same function as _unpack_wheel in py_modules? We should try to avoid duplicating code if possible.  Maybe it can be a standalone util function instead of a member function of the class

Reply:
Yep, will do